tests/order_modifiers_integration/Fees/Create_Fees_Modifiers_Test.php

```diff
@@ -85,45 +85,13 @@ protected function get_table_display_with_search(): string {
 		$_REQUEST = [
 			'modifier' => $this->modifier_type,
 			's'        => 'Unique Fee',
-			'paged'    => 3,
+			'paged'    => 1,
```

Comment:
We shouldn't be changing this test. The point is to test against a previously-identified scenario where incorrect results were returned. Testing for the first page of results is fine, but querying for the 3rd page should actually return the first page of results. That's why `paged` is `3` here.